docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
+
+## Testing your credentials
+
+Before moving to REANA, check that your credentials are in order.
+
+```console
+# Login to lxplus
+$ ssh johndoe@lxplus.cern.ch
+# Make sure userkey.pem is read/write only by the owner
+$ ls -l ~/.globus
+-rw-r--r--. usercert.pem
+-r--------. userkey.pem
+
+# Let us create a proxy certificate with the VO cms
+$ voms-proxy-init --voms cms
```

Comment:
Should we take for granted that the user is going to have this tool installed? Perhaps we should explain it in the introduction.

Reply:
The VOMS client is available on lxplus however the user does need to download the userkey.pem and usercert.pem themselves! I can explain it in the introduction or I can link to a page like this https://ca.cern.ch/ca/Help/?kbid=024010